linkml/generators/sqlalchemygen.py

```diff
@@ -119,6 +126,9 @@ def generate_sqla(
             mappings=tr_result.mappings,
             backrefs=backrefs,
             classname=camelcase,
+            schemaname=underscore,
```

Comment:
We are actualy using a even snakecase function here, but I guess the underscore function is enough for now